tests/mocks/woo-test-helper/woo-test-helper.php

```diff
@@ -94,3 +94,32 @@ function delete_terms_page() {
 	$data  = array( 'post_title' => 'Terms & Conditions' );
 	$wpdb->delete( $table, $data );
 }
+
+/**
+ * Define URL endpoint for setting up cross-sells products.
+ */
+function woocommerce_setup_cross_sells_products() {
+	// phpcs:disable WordPress.Security.NonceVerification.Recommended
+	if ( isset( $_GET['setup_cross_sells'] ) ) {
+		setup_cross_sells();
+		exit( 'Cross-Sells products set up.' );
+	}
+}
+add_action( 'init', 'woocommerce_setup_cross_sells_products' );
+
+/**
+ * Set up Cross-Sells products.
+ */
+function setup_cross_sells() {
+	global $wpdb;
+
+	// phpcs:disable WordPress.DB.PreparedSQL.NotPrepared
+	$select     = "SELECT * FROM {$wpdb->prefix}posts WHERE post_title = '128GB USB Stick' AND post_status = 'publish' AND post_type = 'product'";
+	$id_product = $wpdb->get_row( $select );
+
+	// phpcs:disable WordPress.DB.PreparedSQL.NotPrepared
+	$select        = "SELECT * FROM {$wpdb->prefix}posts WHERE post_title = '32GB USB Stick' AND post_status = 'publish' AND post_type = 'product'";
+	$id_cross_sell = $wpdb->get_row( $select );
+
+	add_post_meta( $id_product->ID, '_crosssell_ids', $id_cross_sell->ID );
+}
```

Comment:
This section adds `32GB USB Stick` as a Cross-Sells product to `128GB USB Stick`. This setup is needed for the following test: https://github.com/woocommerce/woocommerce-blocks/blob/b2014309d9f041bfc1cac140df31efca0414731b/tests/e2e/specs/shopper/cart-checkout/cart.test.js#L116-L121